MacrobenchmarkSample/app/build.gradle.kts

```diff
@@ -84,15 +84,28 @@ dependencies {
     implementation(libs.compose.ui)
     implementation(libs.compose.ui.tooling)
     implementation(libs.constraintlayout)
+    implementation(libs.concurrentfutures)
     implementation(libs.core)
     implementation(libs.datastore)
     implementation(libs.google.material)
     implementation(libs.kotlin.coroutines)
+    implementation(libs.kotlin.coroutines.guava)
     implementation(libs.kotlin.stdlib)
     implementation(libs.lifecycle)
-    implementation(libs.profile.installer)
+    implementation(libs.profileinstaller)
     implementation(libs.squareup.curtains)
     implementation(libs.tracing)
     implementation(libs.viewmodel)
     androidTestImplementation(libs.benchmark.junit)
+
+    // Required until Kotlin version has been upgraded to 1.8.0+.
+    // See: https://youtrack.jetbrains.com/issue/KT-54136/Duplicated-classes-cause-build-failure-if-a-dependency-to-kotlin-stdlib-specified-in-an-android-project
```

Comment:
This is ugly :(

But do we actually need this?
We can remove the stdlib from dependencies https://github.com/android/performance-samples/blob/main/MacrobenchmarkSample/app/build.gradle.kts#L91 (same for `:macrobenchmark` module)

Reply:
I tried and couldn't. So until the Compose compiler supports Kotlin 1.8.0 in an upcoming BOM that's the solution, as presented in the attached link.

Reply:
Oh, it [actually does](https://developer.android.com/jetpack/androidx/releases/compose-kotlin) .. so you can update kotlin to 1.8.10 and compose compiler to 1.4.3 and we're good 🎉

Reply:
🎉